src/providers/outcome-behaviour-map/outcome-behaviour-map.model.ts

```diff
@@ -0,0 +1,12 @@
+export interface FieldDetail {
+  display: string;
```

Comment:
can we make this a string union type if there are just 3 values? i.e.
`display: 'Y' | 'N' | 'A';`

Reply:
Done.